test/e2e/network_test.go

```diff
@@ -27,7 +27,9 @@ func testNetwork(t *testing.T) {
 
 var (
 	// windowsServerImage is the name/location of the Windows Server 2019 image we will use to test pod deployment
-	windowsServerImage = "mcr.microsoft.com/windows/servercore:ltsc2019"
+	// mcr.microsoft.com/windows/nanoserver:1809 do not have Powershell by default. So we are using a hybrid image
+	// which has nanoserver:1809 with the Powershell Core
+	windowsServerImage = "mcr.microsoft.com/powershell:lts-nanoserver-1809"
```

Comment:
I hope the method which returns latest Windows AMI would always return baseOS compatible with the given container image?

https://github.com/openshift/windows-machine-config-bootstrapper/blob/dd78396c3ed478535def2e4635467ef68aa18c67/tools/windows-node-installer/pkg/cloudprovider/aws/ec2.go#L625

Reply:
I will add a comment in `ec2.go` to check for image compatibility

Reply:
I would refrain from adding TODO's to WNI if that what you are saying you will do.
A comment here would be more appropriate saying that 1809 is the Windows build number for Windows Server 2019 LTSC, and that the test image build number has to match the host OS build number.